odra-macros/src/test_utils.rs

```diff
@@ -101,6 +101,29 @@ pub mod mock {
         ModuleImplIR::try_from((&attr, &module)).unwrap()
     }
 
+    pub fn module_invalid_delegation() -> ModuleImplIR {
+        let module = quote! {
+            impl Erc20 {
+                /// Returns the total supply of the token.
+                pub fn total_supply(&self) -> U256 {
+                    self.total_supply.get_or_default()
+                }
+
+                delegate! {
```

Comment:
What is exactly invalid with this code?

Reply:
the `pub` token in `pub fn get_owner(&self) -> Address;`

Reply:
It cannot be handled in macro? 
Stan had a problem with hanging `>` somewhere in the delegate,  it gave no errors - for me this is more invalid.

Reply:
It does not matter if is an extra `pub` or a hanging `>`, it is a syntax error that was suppressed and now is not. The macro expects `syn::TraitFn` or something like that. If cannot parse it, the error is propagated, and the whole `module` macro fails.